coil-core/src/androidMain/kotlin/coil3/decode/StaticImageDecoderDecoder.kt

```diff
@@ -126,6 +127,9 @@ private fun ImageSource.fastImageDecoderSourceOrNull(options: Options): ImageDec
     if (metadata is ResourceMetadata && metadata.packageName == options.context.packageName) {
         return ImageDecoder.createSource(options.context.resources, metadata.resId)
     }
+    if (metadata is ByteBufferMetadata) {
```

Comment:
I think we can add this same code to `AnimatedImageDecoderDecoder`.

Reply:
Done